dynoid/dynoid.go

```diff
@@ -187,10 +195,10 @@ func ReadLocalToken(ctx context.Context, audience string) (*Token, error) {
 
 // AllowHerokuSpace verifies that the issuer is from Heroku for the given host
 // and space id.
-func AllowHerokuSpace(host string, spaceIDs ...string) IssuerCallback {
+func AllowHerokuSpace(issuerHost string, spaceIDs ...string) IssuerCallback {
```

Comment:
This is also the `HEROKU_HOST`:

```suggestion
func AllowHerokuSpace(herokuHost string, spaceIDs ...string) IssuerCallback {
```